### PR TITLE
[DO NOT MERGE] Switch to the new NBC braching scheme

### PR DESCRIPTION
**Pull request by zah**

### CODE REVIEW

**Comment by arnetheduck** [comment]
we should restructure these a bit at the same time such that we have:

* `stable`: 1 large (2500), 1 small  (2000)
* `testing: 1 large(2500), 3 small (1000), 2 mainnet bootnodes
* `unstable`: 2 large (2500), 4 small (1000)
* `libp2p`: 1 small (1000)

File: ansible/inventory/test
Comment (stefantalpalaru) [inline ansible/inventory/test:66]: You're missing master-small-01 which handles the builds for all \*-small-\* hosts.

File: ansible/master.yml
Comment (stefantalpalaru) [inline ansible/master.yml:11]: I suspect there's a hidden master host group list used by Terraform to generate this file and the group names come from there, so it's something to keep in mind.

File: ansible/inventory/test
Comment (jakubgs) [inline ansible/inventory/test:52]: You can't edit this file by hand. Look at the top:
```
# NOTE: This file is generated by terraform.py
# For emergency use when Consul fails
```

File: ansible/inventory/test
Comment (jakubgs) [reply ansible/inventory/test:66]: You can't edit this file by hand. Look at the top:
```
# NOTE: This file is generated by terraform.py
# For emergency use when Consul fails
```

File: ansible/master.yml
Comment (jakubgs) [reply ansible/master.yml:11]: Yeah. This should probably be set to:
```yaml
  hosts:
    - master-large-01.aws-eu-central-1a.nimbus.pyrmont
    - master-small-01.aws-eu-central-1a.nimbus.pyrmont
```

Comment (jakubgs) [reply ansible/master.yml:11]: The groups are indeed generated from Terraform, as is the inventory file by [`ansible/terraform.py`](https://github.com/status-im/infra-nimbus/blob/master/ansible/terraform.py).

File: ansible/host_vars/master-large-01.aws-eu-central-1a.nimbus.pyrmont.yml
Comment (jakubgs) [inline ansible/host_vars/master-large-01.aws-eu-central-1a.nimbus.pyrmont.yml:20]: I guess `1.0.6` now?

Comment (zah) [reply ansible/host_vars/master-large-01.aws-eu-central-1a.nimbus.pyrmont.yml:20]: Yep

File: ansible/group_vars/nimbus-pyrmont-testing.yml
Comment (arnetheduck) [inline ansible/group_vars/nimbus-pyrmont-testing.yml:9]: shall we move one validator-less node to `unstable` instead?

Comment (arnetheduck) [reply ansible/group_vars/nimbus-pyrmont-testing.yml:9]: ...and run one testing node with a single validator?

File: ansible/inventory/test
Comment (jakubgs) [inline ansible/inventory/test:4]: No, you can't change those files by hand. To rename the hosts and change their groups Terraform configuration has to be changed, which you don't have the keys to do.

**Review by jakubgs — APPROVED** [review]
Looks good, but I'll have to test this locally and apply the terraform changes. Then I'll update this PR and merge afterwards.

File: ansible/group_vars/nimbus-pyrmont-testing.yml
Comment (zah) [reply ansible/group_vars/nimbus-pyrmont-testing.yml:9]: We do have another node running a single validator.

My reasoning here was that these are the bootstrap nodes which have an important function for our users. We want slightly higher stability for them. I picked `testing` because this will give us more value as a canary release before deploying bootstrap node changes to mainnet.

File: ansible/inventory/test
Comment (zah) [reply ansible/inventory/test:4]: Well, let's see if the generator will disagree with the edits I made :P

File: ansible/group_vars/nimbus-pyrmont-testing.yml
Comment (arnetheduck) [reply ansible/group_vars/nimbus-pyrmont-testing.yml:9]: ah ok - yeah, we have a single-validator node, but no _testing_ single-validator node (for a/b of single validator behaviour) - we also have no _unstable_ 0-validator node.

Comment (arnetheduck) [reply ansible/group_vars/nimbus-pyrmont-testing.yml:9]: anyway, these are minor points, whatever works

**Comment by jakubgs** [comment]
Okay, after talking to @zah we are putting this on hold until we have a tool for splitting/merging slashing DBs.

For now I've deployed the new hosts in a separate commit: https://github.com/status-im/infra-nimbus/commit/2b44636c

And I've refreshed this PR to take that into account. We'll let the new nodes without validators sync and migrate afterwards.

**Pull request merged**
